Purchases/Networking/Backend.swift

```diff
@@ -495,14 +495,17 @@ private extension Backend {
         }
 
         do {
-            let customerInfo = try CustomerInfo(data: response)
+            let customerInfo = try parseCustomerInfo(fromMaybeResponse: maybeResponse)
             let created = statusCode == HTTPStatusCodes.createdSuccess.rawValue
             Logger.user(Strings.identity.login_success)
             completion(customerInfo, created, nil)
-        } catch {
+        } catch let customerInfoError {
             Logger.error(Strings.backendError.customer_info_instantiation_error(maybeResponse: response))
+            let extraContext = "statusCode: \(statusCode), json:\(maybeResponse.debugDescription)"
```

Comment:
Including the whole response might leak some personal data here?

Reply:
But that is already what is the code in the `handle()` method does.